challenges/platform/README.md

```diff
@@ -0,0 +1,59 @@
+
+
+## Devops / DevSecOps / System Administrators / Production engineers / etc
+Team members with experience in various operations focused disciplines are focussed on the day to day delivery of platform components, such as build/testing automation of infrastructure as code, deploying and configuring our metrics, alarming and logging systems, automating build pipeliness, and other day to day activities to enable our cloud infrastructure.
+
+
+### Challenge
+We use two interviews for team members with operational backgrounds, the first is a lightweight scripting / programming challenge, and the second is a troubleshooting production test. See the following links for outlines of each challenge.
+1. [Devops development challenge](devops.md)
+2. [Troubleshooting challenge](troubleshooting.md)
+
+### Levels
+
+
+| Level  | Oncall  | Operations  | Security  | Communications / Writing  | Networking  | Systems Engineering  | OS Understanding  | Tooling |
+|---|---|---|---|---|---|---|---|---|
+| 1 | Participates in on-call and able to execute run books | Documents and Executes maintenance activities in production | Able to follow security guidelines and best practices in a production environment  | Contributor to internal documentation such as updating run books | Able to troubleshoot introductory networking issues, network policies, routing, etc. | Focus on and contribute to specific area of production system (such as the deployment of our monitoring stack)  | Working understanding of core OS concepts, filesystem, networking, cgroups, namespaces, security mechanisms, etc.  | Work with team members on deployment and maintenance of internal tooling |
+| 2 | Can investigate and solve a majority of production incidents and issues with peer review  | Develops and tests maintenance of production environment (backup/restore as an example)  | Able to analyze and secure new tools or systems connectivity  | Write new documentation, run-books, guides. Able to lead knowledge sharing sessions  |   | Design, write, and test automation for different environments, such as chat bots, upgrade scripts, etc. |   | Able to take ownership and key contributor to key tooling.|
+| 3 | Provide leadership, guidance, and act as escalation point for junior team members |   | Take charge of security incident coordination and create recommendations based on new findings | Contribute to engineering blog, meetups, booth duty, etc to advertise product and engineering culture at Teleport  | Solid understanding of advanced networking, routing, load balancing concepts and scaling production systems  | Design and implement significant automation for challenging problem spaces, such as implementing horizontal/vertical autoscaling and resource allocation | Solid understanding of workload isolation, syscall filtering, linux internals, for high security and production workloads  | Take design lead on key operational areas, such as introducing new tooling, new monitoring, or complex automation |
```

Comment:
Security:
> Take charge of security incident coordination and create recommendations based on new findings

I want "create recommendations" to be a bit more active ownership at l3.  No "I recommend others do the work" -- we want folks to drive what they believe in to done/good enough.  Perhaps:

> Take charge of security incident coordination, create and execute improvements based on new findings